staff/asier/react-spotify-api-promises/design/scripts/main.js

```diff
@@ -0,0 +1,171 @@
+$(document).ready(function () {
+    "use strict"
+
+    $("#listArtists, #listAlbums, #listSongs, #backToArtists, #error").hide()
+
+
+    const showError = () => {
+        $("#error").empty()
+        $("#error").append('<div class="d-block bg-light p-5"><h1 class="display-1 text-success text-center" >  No artist matching your search! </h1></div>')

```

Comment:
WARN! remember to reuse $ elements when possible:
const $error = $('#error') // define this at the top level scope
...
$error.empty() // use the defined $ element here and reuse it on...
$error.append(...)

Reply:
SAME for the other $ elements ($artistList, $albumList, ...)